.bazelrc

```diff
@@ -1,5 +1,6 @@
 common --enable_platform_specific_config
 common --incompatible_use_plus_in_repo_names
+common --experimental_google_legacy_api # needed by protobuf
```

Comment:
Is this only needed by building Bazel itself? What about other projects using protobuf?

Reply:
this is indeed going to be required for all projects using protobuf 29.0+. I think the plan is to flip this flag on by default in Bazel 8. cc @comius

Reply:
I see, hopefully that could be done. I'm seeing failures related to this in https://github.com/bazelbuild/bazel-central-registry/pull/2869